modules/storages/app/models/storages/storage.rb

```diff
@@ -51,6 +51,8 @@ class Storages::Storage < ApplicationRecord
   has_many :projects_storages, dependent: :destroy, class_name: 'Storages::ProjectStorage'
   # We can get the list of projects with this Storage enabled.
   has_many :projects, through: :projects_storages
+  # A storage authenticates at a storage via OAuth.
```

Comment:
confusing comment: more like `A storage authenticates at its remote origin via OAuth.` You can elaborate even a bit more, to make clear, what is "our" storage and what is "the remote storage".